airflow/plugins/operators/elavon_to_gcs_raw.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import gcsfs
+import paramiko
+
+from airflow.models import BaseOperator
+
+CALITP__ELAVON_SFTP_PASSWORD = os.environ["CALITP__ELAVON_SFTP_PASSWORD"]
+BIGQUERY_KEYFILE_LOCATION = os.environ["BIGQUERY_KEYFILE_LOCATION"]
```

Comment:
I don't think this exists in non-PodOperators; the Composer workers are already authenticated to GCP

Reply:
Resolved in [e940285](https://github.com/cal-itp/data-infra/pull/2376/commits/e9402850603a43356d25c614b7a077d2c8f52be1)